metadata-ws/src/test/resources/features/security.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Security related tests
+
+  Scenario: access security disabled links
+    When I request unauthorized GET /
+    Then the response code should be 200
+    When I request unauthorized GET /swagger-ui.html
+    Then the response code should be 200
+
+
+  Scenario: update security enabled links
+    When I request POST /publications with JSON payload:
```

Comment:
there is no "Given" in this? like which user and what roles he has ??

Reply:
Will include `Given`.

I have retained all statements in the feature files as it is. Implementation java files have: 
1. `testoperator` with ROLE_SERVICE_OPERATOR authority for all POST, PUT, PATCH and DELETE operations.
2. `testuser` with default authority for GET operations
I will have to change statements in places, ofcourse it can be done. Please let me know.

3. Where there are negative test cases i have explicitly mentioned `unauthorized` or `less privileges` mean no or lower authority.